### ISSUE TITLE
Helm install fails when applying due to service account

### ISSUE THREAD

**Issue by saminahbab** [spotify/flink-on-k8s-operator#182]
When installing the helm chart using the command: 

```
helm install flink flink-operator-repo/flink-operator --set operatorImage.name=gcr.io/flink-operator/flink-operator:latest

```
I get the following error: 

```
+ kubectl -n flink-operator-system apply -f -
Error from server (Forbidden): error when retrieving current configuration of:
Resource: "/v1, Resource=secrets", GroupVersionKind: "/v1, Kind=Secret"
Name: "webhook-server-cert", Namespace: "flink-operator-system"
Object: &{map["apiVersion":"v1" "data":map["tls.crt":"LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUM5RENDQWR3Q0ZESktNS1lRcEc5L29GNGhEdGMxbEdvZkVCYTlNQTBHQ1NxR1NJYjNEUUVCQ3dVQU1Db3gKS0RBbUJnTlZCQU1NSDBGa2JXbHpjMmx2Yml
from server for: "STDIN": secrets "webhook-server-cert" is forbidden: User "system:serviceaccount:flink-operator-system:default" cannot get resource "secrets" in API group "" in the namespace "flink-operator-system .. 

```

above logging is truncated

It looks like the service account for the the init job for certs is not configured correctly for GET and CREATE of secrets?

**Comment by regadas** [comment]
i belive this might be covered by #180

**Comment by regadas** [comment]
Closed via #180
